src/Otiport.API/Services/IMedicineService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Otiport.API.Contract.Request.Medicines;
+using Otiport.API.Contract.Response.Medicines;
+
+namespace Otiport.API.Services
+{
+    public interface IMedicineService
+    {
+        Task<AddMedicineResponse> AddMedicineAsync(AddMedicineRequest request);
+        Task<DeleteMedicineResponse> DeleteMedicineAsync(DeleteMedicineRequest request);
+        Task<GetMedicinesResponse> GetMedicinesAsync();
```

Comment:
Get de Request class ını almalı. Örneğin burada bir pagination gitmesi gerekebilir. Burada da o parametreleri alıp işlenmesi sağlanabilir